selftests/functional/test_loader.py

```diff
@@ -215,6 +215,9 @@ def test_load_not_a_test(self):
     def test_load_not_a_test_not_exec(self):
         self._test('notatest.py', NOT_A_TEST, 'NOT_A_TEST')
 
+    @unittest.skipIf(os.environ.get("AVOCADO_CHECK_FULL") != "1",
+                     "Skipping test that take a long time to run, are "
+                     "resource intensive or time sensitve")
```

Comment:
The executed test is `print "ahoj"` followed by `assertTrue(True)`. Do you consider this as a time-sensitive test? Basically we used to have a fork-bomb-bug when executing SIMPLE test and we used 5s as a safe deadline to avoid filling the memory. If it's causing troubles we should increase the timeout (and risk OOM killer) rather than disabling this test as it is quite useful and with the bug fixed it's not sensitive at all.

So once again, it's not time, nor memory sensitive test unless there is a breakage. Then it'll work as a fork bomb and to avoid OOM we used 5s deadline, which is apparently to short.